src/lib/createGHA/index.ts

```diff
@@ -186,7 +191,7 @@ export default async function createGHA(
   if (msg) info(msg, false);
 
   if (opts.github) {
-    info(chalk.bold("\n🚀 Let's get you set up with GitHub Actions! 🚀\n", false));
+    info(chalk.bold("\n🚀 Let's get you set up with GitHub Actions! 🚀\n"), false);
```

Comment:
Might be worth changing the signature on `info` to have that `includeEmojiPrefix` option be supplied as an object instead to prevent this from happening again